inference/marian/scripts/alphas/extract_stats.py

```diff
@@ -1,63 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
This one is technically used in the pipeline, but I'll restore it in a later PR where we use this code in the training pipeline. The training pipeline is still building from browsermt/marian-dev.